### PR TITLE
Add open source license

### LINKED ISSUE #13: Add open source license
I think a MIT license should be fine for this project, do you agree? If yes, I can open a PR adding it! 🙂

### PR DESCRIPTION
**Pull request by AlbertoPdRF**
Fixes #13.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Lakshya Khatri
+
```
Comment (LakshyaKhatri) [inline LICENSE]: Please add your names too here

**Review by LakshyaKhatri — CHANGES_REQUESTED** [review]
Kindly make these changes

File: LICENSE (hunk 1 above)
Comment (AlbertoPdRF) [reply LICENSE]: Done!

**Review by LakshyaKhatri — APPROVED** [review]
Perfect! :tada: . Merging.

**Pull request merged**